src/kani-compiler/Cargo.toml

```diff
@@ -12,27 +12,23 @@ atty = "0.2.14"
 bitflags = { version = "1.0", optional = true }
 cbmc = { path = "../../cprover_bindings", package = "cprover_bindings", optional = true }
 clap = "2.33.0"
-cstr = { version = "0.2", optional = true }
 kani_queries = {path = "kani_queries"}
 kani_metadata = { path = "../kani_metadata", optional = true }
 libc = { version = "0.2", optional = true }
-measureme = { version = "9.1.0", optional = true }
 num = { version = "0.4.0", optional = true }
 rustc-demangle = { version = "0.1.21", optional = true }
 serde = { version = "1", optional = true }
 serde_json = { version = "1", optional = true }
 shell-words = "1.0.0"
-smallvec = { version = "1.6.1", features = ["union", "may_dangle"], optional = true }
-snap = { version = "1", optional = true }
+#smallvec = { version = "1.6.1", features = ["union", "may_dangle"], optional = true }
```

Comment:
Is this no longer needed? If so, should we delete it?

Reply:
oops. leftover from me trying to remove them speculatively to find the unused ones. removed that line.